frontend/src/components/cards/cases-per-day-card/cases-per-day-card.jsx

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
 import ReactEcharts from 'echarts-for-react';
+import { Tabs } from '../../layout/tabs/tabs';
 import { Card } from '../../layout/card/card';
 import { Constants } from '../../../config/globals';
 
+const tabList = ['Creștere zilnică', 'Creștere exponențială'];
```

Comment:
Would change from `exponențială` to `cumulativă`
Also, change `exponential` to `cumulative` across all the relevant components

Reply:
fixed, thank you